File: Plugins/Aspose_Slides_Java_for_PHP/src/aspose/slides/WorkingWithSlidesInPresentation/Hyperlinks.php
<?php

namespace Aspose\Slides\WorkingWithSlidesInPresentation;

use com\aspose\slides\Presentation as Presentation;
use com\aspose\slides\SaveFormat as SaveFormat;


class Hyperlinks {

    public static function run($dataDir=null)
    {

        # Instantiate Presentation class that represents the presentation file
        $pres = new Presentation($dataDir . 'demo.pptx');

        # Removing the hyperlinks from presentation
        $pres->getHyperlinkQueries()->removeAllHyperlinks();

        # Saving the presentation
        $save_format = new SaveFormat();
        $pres->save($dataDir . "Hyperlinks.pptx", $save_format->Pptx);

        print "Removed hyperlinks successfully, please check the output file." . PHP_EOL;
    }

}

File: Plugins/Aspose-Slides-Java-for-Jython/asposeslides/WorkingWithSlidesInPresentation/Hyperlinks.py
from asposeslides import Settings
from com.aspose.slides import Presentation
from com.aspose.slides import SaveFormat

class Hyperlinks:

    def __init__(self):
        dataDir = Settings.dataDir + 'WorkingWithSlidesInPresentation/Hyperlinks/'    
        
        # Instantiate Presentation class that represents the presentation file
        pres = Presentation(dataDir + 'demo.pptx')

        # Removing the hyperlinks from presentation
        pres.getHyperlinkQueries().removeAllHyperlinks()

        # Saving the presentation
        save_format = SaveFormat
        pres.save(dataDir + "Hyperlinks.pptx", save_format.Pptx)

        print "Removed hyperlinks successfully, please check the output file."

if __name__ == '__main__':        
    Hyperlinks()